apps/src/code-studio/pd/regional_partner_mini_contact/RegionalPartnerMiniContact.jsx

```diff
@@ -139,3 +167,101 @@ export default class RegionalPartnerMiniContact extends React.Component {
     }
   }
 }
+
+export class RegionalPartnerMiniContactPopupLink extends React.Component {
+  static propTypes = {
+    zip: PropTypes.string,
+    notes: PropTypes.string,
+    sourcePageId: PropTypes.string.isRequired,
+    children: PropTypes.node.isRequired
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showing: false,
+      options: null
+    };
+
+    $.ajax({
```

Comment:
It looks like we've potentially got two of these components on one page, which would mean two calls to this API when just one would do.  Not sure it's worth optimizing for now, but this seems like data that's unlikely to change often and would be nice to move to a static helper and cache so that we only did this once per page, no matter how many links we included.

Reply:
Yes, I did notice that but similarly figured it was rare enough that we didn't need to optimise at this stage.